PennMobile/Course Schedule/CourseScheduleViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  CourseScheduleViewController.swift
+//  PennMobile
+//
+//  Created by CHOI Jongmin on 14/2/2020.
+//  Copyright © 2020 PennLabs. All rights reserved.
+//
+
+import UIKit
+
+protocol CourseScheduleCellDelegate: ModularTableViewCellDelegate, BuildingMapSelectable {}
+
+class CourseScheduleViewController: GenericViewController, IndicatorEnabled, ShowsAlert {
```

Comment:
Let's move the view controller and model to the `Course` folder rather than creating a new folder. It makes sense to put it there.

Reply:
Just committed the directory change. Should I squash and merge the branch into the development?